css/styles.css

```diff
@@ -385,3 +421,69 @@ footer .logo {
   background-color: var(--batt-prim-bg-color);
   color: var(--white-text-color);
 }
+
+/* logo (svg-sprite) */
+/* Contacts */
+.contacts-logo {
+  margin-right: 10px;
+  fill: currentColor;
+}
+
+.button-logo {
+  width: 170px;
+  height: 92px;
+
+  padding: 0;
+  margin: 0;
+  color: var(--logo-color);
+  border: 1px solid var(--logo-border);
+  background-color: var(--primary-background-color);
+}
+
+.button-logo:hover {
+  color: var(--accent-text-color);
+  border: 1px solid var(--accent-text-color);
+}
+.icon-logo {
+  fill: currentColor;
+}
+
+.benefits .benefits-logo {
+  display: block;
+  width: 270px;
+  height: 120px;
+  background-color: var(--batt-sec-bg-color);
+  border: 1px solid transparent;
+  border-radius: 4px;
+  padding: 25px 100px;
+  margin-bottom: 30px;
+}
+
+.team .media {
+  display: flex;

```

Comment:
You have already this property for .team .list
Also you should wrap all text under the image in div add paddings on it. Because in case of long name in will stick to card border